monopoly-project-template/index.css

```diff
@@ -240,8 +266,7 @@
 
 .way__col .house {
   width: 100%;
-  height: 20%;
```

Comment:
Đây cũng thế

Reply:
House's height and width fixed. Add slot inf template and game inf. Please check!